derive/src/to_delegate.rs

```diff
@@ -244,11 +244,10 @@ impl RpcMethod {
 				quote! { let params: _jsonrpc_core::Result<_> = Ok((params,)); }
 			} else if self.attr.params_style == Some(ParamStyle::Positional) {
 				quote! { let params = params.parse::<(#(#param_types, )*)>(); }
-			} else
-			/* if self.attr.params_style == Some(ParamStyle::Named) */
-			{
+			} else {
 				unimplemented!("Server side named parameters are not implemented");
 			}
+			/* if self.attr.params_style == Some(ParamStyle::Named) */
```

Comment:
Looks kind of weird now, perhaps move it above `unimplemented!` call?

Reply:
Should we not simply remove this comment ? It is 13 months old.
Do you remember the purpose of keeping it ?

Reply:
Nope, not really :) Yeah, let's remove it. If we wanted to be explicit about the variant that is handled here, it would be better to use `match` instead.